paseos/visualization/space_animation.py

```diff
@@ -0,0 +1,205 @@
+import sys
+
+sys.path.append("../")
+
+from paseos.actors.base_actor import BaseActor
+from paseos.actors.ground_station_actor import GroundstationActor
+from paseos.actors.spacecraft_actor import SpacecraftActor
+from paseos.paseos import PASEOS
+import paseos
+from animation import Animation
+
+import pykep as pk
+import numpy as np
+from dotmap import DotMap
+
+import matplotlib.pyplot as plt
+from matplotlib.font_manager import FontProperties
+
+
+class SpaceAnimation(Animation):
```

Comment:
this class needs to be added here for successful import outside paseos https://github.com/aidotse/PASEOS/blob/2f2d7c342ffbb10ce36e0f5727463220725d3631/paseos/__init__.py#L30